api/AudioTrack.json

```diff
@@ -664,7 +664,7 @@
               "version_added": false
             },
             "ie": {
-              "version_added": "10"
+              "version_added": false
```

Comment:
Can you also update api.VideoTrack.sourceBuffer? They should go together, otherwise something odd is happening.

Reply:
Done now, thanks!